src/main/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/AutomatedCaseUpdater.java

```diff
@@ -58,17 +59,29 @@ private void addNewBundle(BundleConfiguration configuration, ArrayNode bundles)
         bundle.setHasCoversheets(CcdBoolean.Yes);
         bundle.setHasTableOfContents(CcdBoolean.Yes);
         bundle.setFileName(configuration.filename);
-        int sortIndex = 0;
 
-        for (BundleConfigurationFolder folder : configuration.folders) {
+        addFolders(configuration.folders, bundle.getFolders(), 0);
+
+        bundles.add(bundleDtoToBundleJson(bundle));
+    }
+
+    private int addFolders(List<BundleConfigurationFolder> sourceFolders,
```

Comment:
will this ever get called with sortindex != 0?
maybe in the future situation where docs and folders can be imported by the yaml file

Reply:
it's a recursive funciton... it calls itself and keeps track of the index as it goes